docs/miscellaneous.rst

```diff
@@ -168,14 +168,14 @@ These steps are applied to each basic block and the newly generated code is used
     else
       return 1;
 
-is simplified to code which can also be compiled from
+The process simplifies to code which can also be compiled from:
 
 ::
 
     data[7] = 9;
     return 1;
 
-even though the instructions contained a jump in the beginning.
```

Comment:
Please either keep it as it was or change these into full sentences.

Reply:
@chriseth Try this.